JSDemos/Demos/Lookup/Templates/description.md

```diff
@@ -1 +1,9 @@
-This demo shows how you can customize the input field and the items of&nbsp;the Lookup component.
\ No newline at end of file
+The Lookup contains properties to specify custom templates for the following elements:
+
+* Input field: [fieldTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#fieldTemplate)
+
+* Items: [itemTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#fieldTemplate)
+
+* Group captions: [groupTemplate](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#groupTemplate)
```

Comment:
Please note that React uses different properties. Seems like you need to create a separate description for the React approach.